### PR TITLE
Added Import/Export Area In Block Representations (Copied Code From Scratch-Blocks)

### PR DESCRIPTION
**Pull request by TheBrokenRail**
Yes, I did my own feature request!

### CODE REVIEW

File: playground/index.html [hunk 1]
```diff
@@ -15,7 +15,8 @@
         <p>
             <a id="renderexplorer-link" href="#">Renderer</a><br />
             <a id="threadexplorer-link" href="#">VM Threads</a><br />
-            <a id="blockexplorer-link" href="#">VM Block Representation</a>
+            <a id="blockexplorer-link" href="#">VM Block Representation<br /></a>
```
Comment (tmickel) [inline playground/index.html]: `</a>` before `<br />`

File: playground/index.html [hunk 2]
```diff
@@ -29,6 +30,17 @@
...
             <pre id="blockexplorer"></pre>
         </div>
+        <div id="tab-importexport">
+            Import/Export XML
+            <p>
+                <input type="button" value="Export to XML" onclick="toXml()">
+                &nbsp;
+                <input type="button" value="Import from XML" onclick="fromXml()" id="import">
+                <br><br>
+                <textarea id="importExport" style="width: 480px; height: 360px"
```
Comment (tmickel) [inline playground/index.html]: Could we move this style to playground.css?

File: playground/index.html [hunk 3]
```diff
@@ -741,5 +753,36 @@
...
+          function fromXml() {
+            var input = document.getElementById('importExport');
+            var xml = Blockly.Xml.textToDom(input.value);
+            Blockly.Xml.domToWorkspace(workspace, xml);
+            taChange();
+          }
+    
+          // Disable the "Import from XML" button if the XML is invalid.
+          // Preserve text between page reloads.
+          function taChange() {
```
Comment (tmickel) [inline playground/index.html]: For the sake of simplicity, can we cut out `taChange` and the calls to it above (in `toXml`, `fromXml`, and the `onchange` and `onkeyup` handlers in the HTML?)

**Comment by tmickel** [comment]
Thanks @TheBrokenRail. This looks mostly good. It looks to me like the functionality of `taChange` is not working (checked it out). Rather than fix it, I'd be happy to strip it out. Made a couple of style requests, too. If you wouldn't mind making the tweaks, I'd be happy to merge this.

**Comment by TheBrokenRail** [comment]
Did Everything except the move to css. What do you mean by move the xml stuff to css

**Comment by tmickel** [comment]
Not a problem. I meant moving the inline CSS `style="..."` to the external CSS file. But this is close enough, I can tweak it. Thanks again!!

**Pull request merged**